attributes/default.rb

```diff
@@ -10,30 +10,33 @@
   default["sensu"]["windows"]["package_options"] = nil
   default["sensu"]["windows"]["install_dotnet"] = true
   default["sensu"]["windows"]["dotnet_major_version"] = 4
+  default["sensu"]["version"] = "1.0.2-1-x64"
+  default['sensu']['windows_package_version'] = '1.0.2.1'
 else
   default["sensu"]["admin_user"] = "root"
   default["sensu"]["directory"] = "/etc/sensu"
   default["sensu"]["log_directory"] = "/var/log/sensu"
+  default["sensu"]["version"] = "1.0.2-1"
 end
 
 # installation
-default["sensu"]["version"] = "0.28.4-1"
 default["sensu"]["version_suffix"] = nil
 default["sensu"]["apt_repo_codename"] = nil
 default["sensu"]["yum_repo_releasever"] = nil
 default["sensu"]["use_unstable_repo"] = false
 default["sensu"]["log_level"] = "info"
+default["sensu"]["log_rotate_file_size"] = '10240'
+default["sensu"]["log_rotate_file_keep"] = '10'
 default["sensu"]["use_ssl"] = true
 default["sensu"]["use_embedded_ruby"] = true
 default["sensu"]["service_max_wait"] = 10
 default["sensu"]["directory_mode"] = "0750"
 default["sensu"]["log_directory_mode"] = "0750"
 default["sensu"]["client_deregister_on_stop"] = false
 default["sensu"]["client_deregister_handler"] = nil
-
+default["sensu"]["msi_repo_url"] = "http://repositories.sensuapp.org/msi/2016"
```

Comment:
This is a breaking change and needs to be called out in the changelog. Again I would prefer we limit the changes strictly to what is required to use logrotate.

Reply:
The url wihtout the 2016 has no MSI above 0.26.5 :( so I had to change it. I expect but have not tested it that the msi's in the 2016 folder are either backwards compatible or the same as in the 2012R2 folder.

Reply:
On the repository server the 2016 and 2012r2 directories are symlinks to the 2012 directory. There's no difference in the content being served.